src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -2622,6 +2622,68 @@ public void ParseReferences()
             // TODO: multiple files, quotes, etc.
         }
 
+        [Fact]
+        public void ParseReferencesAlias()
+        {
+            Assert.True(parseRef(@"/r:a=util.dll") is
+            {
+                Reference: @"util.dll",
+                Properties.Aliases: ["a"],
+                Properties.EmbedInteropTypes: false
+            });
+
+            Assert.True(parseRef(@"/r:""a=util.dll""") is
+            {
+                Reference: @"a=util.dll",
+                Properties.Aliases: [],
+                Properties.EmbedInteropTypes: false
+            });
+
+            Assert.True(parseRef(@"/r:""c:\users\app=exe\util.dll""") is
+            {
+                Reference: @"c:\users\app=exe\util.dll",
+                Properties.Aliases: [],
+                Properties.EmbedInteropTypes: false
+            });
+
+            Assert.True(parseRef(@"/r:a=b=util.dll") is
```

Comment:
Consider also testing
```
/r:"a=b"=util.dll
/r:a"b=util.dll"
/r:"a"="util.dll"

Reply:
Unescaped quotes are just deleted by the command line parser (historical reasons I cant' explain. So in these cases the quotes don't really impact the output. Not sure if you were aware of that or not when posting the cases here. Added both these versions and the ones where quotes are escaped.